docs/README_Cross-Distro_bootstrap.md

```diff
@@ -0,0 +1,211 @@
+      -C <URL_OF_AWSCLIv1_BUNDLE> \
+      -c <URL_OF_AWSCLIv2_BUNDLE> \
+      -s <URL_OF_AWS_SSM_AGENT_RPM> \
+      -m /mnt/ec2-root
+    ~~~
+    This will typically look something like:
+    ~~~
+    ./AWSutils.sh -d ~/RPM/Amazon/ \
+      -C https://s3.amazonaws.com/aws-cli/awscli-bundle.zip \
+      -c https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip \
+      -s https://s3.us-east-1.amazonaws.com/amazon-ssm-us-east-1/latest/linux_amd64/amazon-ssm-agent.rpm \
+      -m /mnt/ec2-root
+    ~~~
+1. Apply SELinux labels, install GRUB2 stuff, etc., by executing:
+    ~~~
+    ./PostBuild.sh -f xfs -m /mnt/ec2-root -X -z <PREFERRED_TIMEZONE>
+    ~~~
+    This will typically look something like:
+    ~~~
+    ./PostBuild.sh -f xfs -m /mnt/ec2-root -X -z UTC
+    ~~~
+    Note: To ensure that the resultant AMI is _not_ FIPS-enabled, add the `--no-fips` long-option. Similarly, to ensure that `/tmp` is not set up as a `tmpfs` pseudo-filesyste, add the `--no-tmpfs`long-option.
+1. Unmount the disk by executing:
+    ~~~
+    ./Umount -c /mnt/ec2-root
+    ~~~
+
+## Create Image
+
+Once the above, `chroot()`-style build to the secondary volume has successfully completed, you are ready to create an AMI from that secondary disk:
+
+1. Using the AWS CLI or web-console, Snapshot the secondary volume
+1. Create a suitable `blockmap.json` template-file. Contents should look like:
+    ~~~
+    [
+        {
+            "DeviceName": "/dev/sda1",
+            "Ebs": {
+                "DeleteOnTermination": true,
+                "SnapshotId": "__SNAPSHOT_ID__",
+                "VolumeSize": __SIZE__,
+                "VolumeType": "gp2"
+            }
+        }
+    ]
+    ~~~
+    Note: `gp2` is the "normal" value. However, any EBS-type that's valid for the partition/region can be substituted. Simply be aware that select other than `gp2` may limit the instance-types that the resultant AMI can be launched as.
+1. Monitor the snapshot's creation.
+1. When the snapshot completes, register an image of it with something like:
+    ~~~
+    aws ec2 register-image \
+      --virtualization-type hvm \
+      --architecture x86_64 \
+      --ena-support \
+      --sriov-net-support simple \
+      --root-device-name /dev/sda1 \
+      --block-device-mappings "$(
+          sed -e 's/__SIZE__/8/' -e 's/__SNAPSHOT_ID_/<SNAP_ID_FROM_1>/' /root/blockmap.json
+        )" \
+      --name "MyBootstrapAMI-YYYY.MM.N.x86_64-gp2" \
+      --description "\"boostrap\" image for Rocky Linux 8.5 (current through 2021-12-21)"
+    ~~~
+
+## Validate Image
+
+While the above processes have been shown to reliably work for:
+
+* Creating a CentOS 8 Core (now deprecated)
+* Creating a CentOS 8 Stream
+* Creating an Alma Linux 8
+* Creating a Rocky Linux 8
+
+From an official Red Hat 8 AMI from the AWS MarketPlace, it is still a good idea to validate your results before trying to use your bootstrap AMI for AMI-building or other purposes. To do so:
```

Comment:
This introductory phrase seems out of place here....

```suggestion
It is still a good idea to validate your results before trying to use your bootstrap AMI for AMI-building or other purposes. To do so:
```

Reply:
Chose different way to clarify.